connect/eaas/runner/helpers.py

```diff
@@ -86,3 +92,40 @@ def to_ordinal(val):
     if val > 14:
         return f"{val}{ORDINAL_SUFFIX.get(int(str(val)[-1]), 'th')}"
     return f"{val}{ORDINAL_SUFFIX.get(val, 'th')}"
+
+
+def get_connect_version():
+    env = get_environment()
+    url = f'https://{env["api_address"]}/public/v1/accounts'
+    headers = {'Authorization': 'ApiKey MODULE:{}'.format(env['api_key'])}
+
+    connect_response = requests.get(
+        url,
+        headers=headers,
+    )
+    if connect_response.status_code >= 500:
+        logger.warning('Can not retrieve information about Connect version')
+        sys.exit(1)
+
+    if connect_response.status_code != 200:
+        logger.warning(f'API Key is not valid: {connect_response.text}')
+        sys.exit(2)
+
+    return connect_response.headers['Connect-Version']
+
+
+def get_pypi_runner_minor_version(major_version):
+    res = requests.get(PYPI_EXTENSION_RUNNER_URL)
+    if res.status_code != 200:
+        raise requests.exceptions.RequestException(
```

Comment:
Personally I don't like to raise 3rd party exceptions.
I see here to options:

1) call response.raise_for_status() and capture it in the calling funcion
2) raise custom exception
3) raise exception from python stdlib like RuntimeError